includes/multi-currency/Analytics.php

```diff
@@ -528,4 +551,13 @@ private function set_sql_replacements() {
 			],
 		];
 	}
+
+	/**
+	 * Checks whether Custom Order Tables are enabled.
+	 *
+	 * @return bool
+	 */
+	private function is_cot_enabled() {
```

Comment:
Nitpick: can we add a return type hint on this function?

Reply:
Done in https://github.com/Automattic/woocommerce-payments/pull/4677/commits/ed2d295ddd4a19369f85f8896e05f09464071527